src/ansible_creator/cli.py

```diff
@@ -143,9 +143,27 @@ def parse_args(self: Cli) -> argparse.Namespace:
 
         init_command_parser.add_argument(
             "collection",
+            nargs="?",
             help="The collection name in the format ``<namespace>.<collection>``.",
         )
 
+        init_command_parser.add_argument(
+            "--project",
+            default="collection",
+            help="Project type to scaffold, for eg: ansible-project.",
+        )
+
+        init_command_parser.add_argument(
+            "--scm-org",
+            help="Namespace for the scaffolded playbook adjacent collection in ansible-project.",
```

Comment:
The SCM org where the ansible-project will be hosted. This value is used as the namespace for the playbook adjacent collection.

Reply:
"Required when `--project=ansible-project`"